scripts/run_benches_for_pallets.sh

```diff
@@ -19,17 +19,19 @@ pallets=(
 	pallet-web3-names
 	parachain-staking
 	public-credentials
+	pallet-asset-switch
 )
 
 // Add Peregrine-only pallets here!
 if [ "$runtime" = "peregrine" ]; then
   pallets+=(
-	pallet-asset-switch
-  )
+		pallet-bonded-coins
+	)
 fi
 
 echo "[+] Running all default weight benchmarks for $runtime --chain=$chain"
 
```

Comment:
This space is probably not necessary?

Reply:
I fmt the file with [shfmt](https://github.com/mvdan/sh#shfmt) in: [8739367](https://github.com/KILTprotocol/kilt-node/pull/789/commits/87393679e7015a3bea258bf75433596e63e4e7d9)